examples/src/main/scala/eu/neverblink/jelly/examples/PekkoStreamsEncoderFlow.scala

```diff
@@ -1,48 +1,58 @@
-import eu.ostrzyciel.jelly.core.JellyOptions
-import eu.ostrzyciel.jelly.stream.*
+import eu.neverblink.jelly.convert.jena.{JenaAdapters, JenaConverterFactory}
+import eu.neverblink.jelly.core.JellyOptions
+import eu.neverblink.jelly.core.utils.GraphHolder
+import eu.neverblink.jelly.stream.*
 import org.apache.jena.graph.{Node, Triple}
 import org.apache.jena.riot.RDFDataMgr
 import org.apache.jena.sparql.core.Quad
 import org.apache.pekko.actor.ActorSystem
 import org.apache.pekko.stream.scaladsl.*
+import eu.neverblink.jelly.examples.shared.ScalaExample
 
 import java.io.File
 import scala.collection.immutable
-import scala.concurrent.{Await, ExecutionContext}
 import scala.concurrent.duration.*
+import scala.concurrent.{Await, ExecutionContext}
+import scala.jdk.CollectionConverters.*
 
 /**
- * Example of using the [[eu.ostrzyciel.jelly.stream.EncoderFlow]] utility to encode RDF data as Jelly streams.
+ * Example of using the [[eu.neverblink.jelly.stream.EncoderFlow]] utility to encode RDF data as Jelly streams.
  * 
  * Here, the RDF data is turned into a series of byte buffers, with each buffer corresponding to exactly one frame.
  * This is suitable if your streaming protocol (e.g., Kafka, MQTT, AMQP) already frames the messages.
  * If you are writing to a raw socket or file, then you must use the DELIMITED variant of Jelly instead.
- * See [[eu.ostrzyciel.jelly.examples.PekkoStreamsWithIo]] for examples of that.
+ * See [[eu.neverblink.jelly.examples.PekkoStreamsWithIo]] for examples of that.
  *
  * In this example we are using Apache Jena as the RDF library (note the import:
- * `import eu.ostrzyciel.jelly.convert.jena.given`).
+ * `import eu.neverblink.jelly.convert.jena.given`).
  * The same can be achieved with RDF4J just by importing a different module.
  */
-object PekkoStreamsEncoderFlow extends shared.ScalaExample:
+object PekkoStreamsEncoderFlow extends ScalaExample:
   def main(args: Array[String]): Unit =
     // We will need a Pekko actor system to run the streams
     given actorSystem: ActorSystem = ActorSystem()
     // And an execution context for the futures
     given ExecutionContext = actorSystem.getDispatcher
 
+    // We will need a JenaConverterFactory to convert between Jelly and Jena
+    given JenaConverterFactory = JenaConverterFactory.getInstance()
+
+    // We need to import the Jena adapters for Jelly
+    given JenaAdapters.DATASET_ADAPTER.type = JenaAdapters.DATASET_ADAPTER
+    given JenaAdapters.MODEL_ADAPTER.type = JenaAdapters.MODEL_ADAPTER
+
     // Load the example dataset
     val dataset = RDFDataMgr.loadDataset(File(getClass.getResource("/weather-graphs.trig").toURI).toURI.toString)
 
     // First, let's see what views of the dataset can we obtain using Jelly's Iterable adapters:
     // 1. Iterable of all quads in the dataset
-    val quads: immutable.Iterable[Quad] = dataset.asQuads
+    val quads: immutable.Iterable[Quad] = JenaAdapters.DATASET_ADAPTER.quads(dataset).asScala.toList
```

Comment:
This is ugly as all hell, do not subject users to that, please.

Instead, let's use Jena's built-in API – just call `.stream().asScala.toList` or whatever